src/lib/events.js

```diff
@@ -98,10 +120,24 @@ var Events = {
         /*
          * Call all the handlers except the last one.
          */
-        for(var i = 0; i < handlers.length; i++) {
+        for(i = 0; i < handlers.length; i++) {
             handlers[i](data);
         }
 
+        /* Do the same as for external-facing events, except trigger the same
+         * events on the internal handlers. This does *not* affect the return
+         * value. It simply mirrors triggers internally so that there's no
+         * conflict with external user-defined events when plotly manages
+         * events internally.
+         */
+        var internalHandlers = plotObj._internalEv._events[event];
```

Comment:
More specifically, It appear to me that `triggerHandler` was just a more jquery-like version of `emit`, but I could be wrong. If there's no use-case for it, I'm glad to remove. I have no preference of particular knowledge here, so I'll defer to you on this one.